features/support/testing_dsl.rb

```diff
@@ -223,6 +223,13 @@ def add_dep
       end
     end
 
+    class PNPMProject < Project
+      def add_dep
+        add_to_file('pnpm-lock.yaml', '')
+        add_to_file('package.json', '{"dependencies" : {"http-server": "0.11.1"}}')
```

Comment:
This test is failing. Should you be adding this to package.json or to the lock file?